Config.cmake.in

```diff
@@ -0,0 +1,26 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@Targets.cmake")
+
+include(CMakeFindDependencyMacro)
+
+# we do not add the other dependencies because these are header files lib
+if(${CMAKE_VERSION} VERSION_LESS "3.15.0")
```

Comment:
Is this distinction actually needed?  At least according to the documentation, `find_dependency` already exists in 3.10 and built without errors in my workspace when removing the `find_package` case.

Reply:
Yes but it seems that the targets are not properly found form dependencies with this method and in particular the boost targets form the Boost components.